client/src/app/containers/Game/PlayerHand.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Grid from '@material-ui/core/Grid';
+import GridList from '@material-ui/core/GridList';
+import { nanoid } from '@reduxjs/toolkit';
+
+import { Card, PeggedCard, Phase } from './models';
+import PlayingCard from './PlayingCard';
+
+const showOpponentsHand = (phase: Phase) => phase !== 'Deal';
+
+interface Props {
+    hand: Card[];
+    phase: Phase;
+    side?: boolean;
+    pegged: PeggedCard[];
+    mine?: boolean;
+}
+
+const PlayerHand: React.FunctionComponent<Props> = ({
+    hand,
+    phase,
+    side,
+    pegged,
+    mine,
+}) => {
+    if (!hand || !showOpponentsHand(phase)) {
+        return null;
+    }
+
+    return (
+        <Grid
+            item
+            container
+            direction={side ? 'column' : 'row'}
+            justify='center'
+            spacing={1}
+        >
+            <GridList>
+                {hand.map(card => (
+                    <PlayingCard
+                        key={
+                            card.name === 'unknown'
+                                ? `handcard-unknown-${nanoid()}`
+                                : `handcard-${card.name}`
+                        }
+                        card={card}
+                        mine={mine}
+                        disabled={
+                            phase === 'Pegging' &&
+                            pegged &&
+                            pegged.some(pc => pc.card.name === card.name)
```

Comment:
:mag: How is `some` different from `any`? Is `any` a thing?

Reply:
`any` is not a thing. `some` checks if some elements in the array match the condition; not a great name, I agree. Take it up with JS